packages/neutrino-middleware-dev-server/index.js

```diff
@@ -0,0 +1,71 @@
+const ramda = require('ramda');
+const hot = require('neutrino-middleware-hot');
+const opn = require('opn');
+
+module.exports = (neutrino, options = {}) => {
+  if (process.env.NODE_ENV === 'development') {
+    const config = neutrino.config;
+    const server = ramda.pathOr({}, ['options', 'server'], neutrino);
+    const protocol = process.env.HTTPS ? 'https' : 'http';
+    const publicHost = process.env.HOST;
+    const port = process.env.PORT || server.port || options.port || 5000;
+    const https = (protocol === 'https') || server.https || options.https;
+    let openInBrowser = false;
+    let serverPublic = false;
+    let host = 'localhost';
```

Comment:
All this does is use `process.env.HOST` as the highest priority, if `serverPublic` is defined use `0.0.0.0`, and `localhost` as lowest priority.